tests/expose_test.go

```diff
@@ -84,34 +116,72 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		Context("Expose ClusterIP service", func() {
 			const servicePort = "27017"
 			const serviceName = "cluster-ip-vmi"
-			It("[test_id:1531][label:masquerade_binding_connectivity]Should expose a Cluster IP service on a VMI and connect to it", func() {
+
+			var vmiExposeArgs []string
+
+			var job *batchv1.Job
+
+			BeforeEach(func() {
+				vmiExposeArgs = []string{
+					expose.COMMAND_EXPOSE,
+					"virtualmachineinstance", "--namespace", tcpVM.GetNamespace(), tcpVM.GetName(),
+					"--port", servicePort, "--name", serviceName,
+					"--target-port", strconv.Itoa(testPort),
+				}
+			})
+
+			AfterEach(func() {
+				Expect(job).NotTo(BeNil(), "a successful test must have stored a way to delete the batchv1.Job entity")
```

Comment:
The job has [TTLSecondsAfterFinished](https://github.com/kubevirt/kubevirt/blob/d380576d2937b86fcd2cff8a73976fd32186a02e/tests/job.go#L99), afaiu, it means it will be automatically cleaned up.

Do you think this mechanism is not working and we have a leak?

Reply:
It does have, but is defined to 60 seconds. As a result, some of the faster tests will result in a pile of `Job`s.

I'll confirm this TTL mechanism works, and add a method that allows us to specify the TTL, setting it to 0. 

Do you happen to know a reason why we should use a TTL != 0 ? (delete instantly after it fails / completes)

Reply:
I guess to collect logs.

Reply:
We do not collect the `Job`s logs: https://github.com/kubevirt/kubevirt/blob/4a7174ef8907f6eff8e3362bde4fc67030605b38/tests/reporter/kubernetes.go#L103

.... but that doesn't mean we never will. 

Should I just use this version w/ a fixed 60 second TTL ?

Reply:
Since that's how it worked till now, and I'm not aware of any issues with it. I would keep it with the 60 seconds ttl.

Reply:
I agree with this, and right now, we do not even store the logs of jobs.

I personally don't think that there's much chance of the job log actually having anything meaningful / useful for debug - at least the ones we have right now (given their implementation) .

I would personally just delete them, without any regard for their logs.

I can also just follow the current status quo, and do nothing with them, and they will go away on their own, since each alternative has its advantages (doing nothing vs being explicit + making sure there are no leaks) . 

Which of the 2 should I implement ? @EdDev / @AlonaKaplan